src/index.js

```diff
@@ -0,0 +1,45 @@
+// eslint-disable-next-line
+import _ from 'lodash';
+import './style.css';
+
+import { list, ItemSaved } from './modules/addRemoveItems.js';
+import itemsList from './modules/createNewItems.js';
+
+const listForm = document.querySelector('#todo-form');
+const taskList = document.querySelector('.task-list');
+const taskCompleteButton = document.querySelector('#delete-tasks');
+const addTask = document.querySelector('#list-input');
+
+listForm.addEventListener('submit', (e) => {
+  e.preventDefault();
+  const savedTask = new ItemSaved();
+  savedTask.newItem(addTask.value);
+  listForm.reset();
+});
+
+for (let i = 1; i <= list.length; i += 1) {
+  list.forEach((lis) => {
+    if (lis.index === i) {
+      itemsList(lis);
+    }
+  });
+}
+
+// clear all completed tasks
+taskCompleteButton.addEventListener('click', () => {
+  let dolist = list.filter((element) => element.completed === false);
+
+  // code for updating index
+  let i = 1;
+  dolist = dolist.map((element) => {
+    element.index = i;
+    i += 1;
+    return element;
+  });
+
+  taskList.innerHTML = '';
+  dolist.forEach((itemList, index) => {
+    itemList(itemList, index + 1);
+  });
+  localStorage.setItem('list', JSON.stringify(dolist));
+});
```

Comment:
- [ ] Please check the implementation of the button click for clearing completed tasks. A click on the button removes all items from the DOM. 😨 

I suggest you use the following steps to solve this problem. 🧰 
- Check for Local Storage for all Objects with completed properties = false.
- implement a filter() function for the Local Storage array to return only objects with completed to false. 
- save to Local storage and render to DOM.

You can refer  [here](https://www.w3schools.com/jsref/jsref_filter.asp) on how to use Array filter(). 👍🏿